src/shared/lib/regex.test.ts

```diff
@@ -0,0 +1,29 @@
+import { it, describe, expect } from "vitest";
+
+import { numbersGroupingRegex } from "./regex";
+
+describe("numbersGroupingRegex", () => {
+  it("int numbers", () => {
```

Comment:
It's better to define a group of tests with `describe` function and don't have `it` inside `it`.